lib/qesapdeployment.pm

```diff
@@ -244,6 +245,36 @@ sub qesap_get_inventory {
     return "$paths{deployment_dir}/terraform/" . lc $provider . '/inventory.yaml';
 }
 
+=head3 qesap_get_agents_number
+
+Get the number of SAP cluster nodes where the trento-agent has been installed
+=cut
+
+sub qesap_get_agents_number {
```

Comment:
Wouldn't it be better to have this function outside of the deployment library since it is trento related?

Reply:
I can change the description to 

```
Get the number of cluster nodes
```

and even the function name to **qesap_get_nodes_number**

The code is not Trento related. The implementation is explained in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15797/files#diff-c4f6960b011dade32b03b0797381a30cb073505ec0cab10ca6c82fdcaba36a6bR177

It is just an inventory.yaml parser that return the number of nodes

Reply:
Yeah renaming it to "qesap_get_nodes_number" sounds good.